README.md

```diff
@@ -1 +1,73 @@
-wordle-dictionary-parser
+# wordle-dictionary-parser
+## Usage
+Includes usage and examples of included scripts.
+### **parse_dictionary.py**
+Parses out and separates words based on lengths and then writes them to .json files.
+#### Usage
+`usage: parse_dictionary.py [-h] -i INPUT -o OUTPUT [-fn FILENAME] [--min MIN] [--max MAX] [-t THRESHOLD] [-c]`
+```
+Outputs a .json file of words based on length and word frequency from a dictionary .json file.
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -i INPUT, --input INPUT
+                        Input file (.json dictionary). Required
+  -o OUTPUT, --output OUTPUT
+                        Output file directory. Required
+  -fn FILENAME, --filename FILENAME
+                        Output file name.
+  --min MIN             Minimum word length to be parsed into a file.
+  --max MAX             Maximum word length to be parsed into a file.
+  -t THRESHOLD, --threshold THRESHOLD
+                        Zipf frequency threshold to use if parsing out hard words.
+  -l LANG, --lang LANG  language of the word being checked (default: english 'en')
+  -c, --combine         Parse out words of same length into single file. I.e. don't parse out 'hard' words.
+```
+#### Examples
+```
+python3 parse_dictionary.py -i ./data/dictionary_compact.json -o ./data/
+```
+Output files:
+- 4_letters_hard.json
+- 4_letters_normal.json
+- 5_letters_hard.json
+- 5_letters_normal.json
+<br />**...**
```

Comment:
If you're trying to put a markdown 'newline', it's just an empty line.  VS Code has a "Preview Markdown" tool built in which can help with this.

Reply:
Yeah I was trying to keep the ellipses in line with the bullet points for formatting purposes because I didn't like how it looked with a bigger line break. I have used VS Code's markdown preview it has been very helpful!